yt/visualization/volume_rendering/scene.py

```diff
@@ -526,7 +526,9 @@ def _show_mpl(self, im, sigma_clip=None, dpi=100):
         from matplotlib.figure import Figure
 
         s = im.shape
-        self._render_figure = Figure(figsize=(s[1] / float(dpi), s[0] / float(dpi)))
+        self._render_figure = Figure(
+            figsize=(s[1] / float(dpi), s[0] / float(dpi)), dpi=dpi
```

Comment:
note, this is the actual fix. the default figure dpi was changing due to the firefly import (a question for firefly folks?) and so this Figure needs to be instantiated with the `dpi` variable value